integration-tests/testutils/common/common.go

```diff
@@ -39,8 +39,9 @@ import (
 const (
 	// BaseAltPartitionPath is the path to the B system partition.
 	BaseAltPartitionPath = "/writable/cache/system"
-	needsRebootFile      = "/tmp/needs-reboot"
-	channelCfgFile       = "/etc/system-image/channel.ini"
+	// NeedsRebootFile is the file that a test writes in order to request a reboot.
+	NeedsRebootFile = "/tmp/needs-reboot"
```

Comment:
This seems to work only if I'm in the same package of the export. As here I'm trying to export a var from the common package to be used in a test of the report package, it's not finding it. Unless I'm doing something wrong...